duckdb-latest/setup-duckdb-latest.sh

```diff
@@ -5,12 +5,12 @@ set -e
 mkdir -p ./duckdb-latest/r-duckdb-latest
 # Rscript -e  'withr::with_libpaths(new = "./duckdb-latest/r-duckdb-latest", devtools::install_github("duckdb/duckdb/tools/rpkg"))'
 # prevent errors when running 'ver-duckdb.sh'
-Rscript -e 'install.packages("DBI", lib="./duckdb-latest/r-duckdb-latest")'
+Rscript -e 'install.packages("DBI", lib="./duckdb-latest/r-duckdb-latest", repos = "http://cran.us.r-project.org")'
```

Comment:
it is generally better to use `cloud.r-project.org` so other regions will not have to via us mirror